ComputeClient/Compute.Client.UnitTests/Snapshot/SnapshotAccessorTests.cs

```diff
@@ -76,23 +76,41 @@ public async Task GetSnapshotServicePlansPaginated_ReturnsResponse()
 		}
 
 		[TestMethod]
-		public async Task GetSnapshotServicePlansPaginated_WithFiltersAndPagingOptions()
+		public async Task ListSnapshotServicePlansPaginated_WithFiltersAndPagingOptions()
 		{
-			requestsAndResponses.Add(ApiUris.ListSnapshotServicePlans(accountId), RequestFileResponseType.AsGoodResponse("ListSnapshotServicePlansResponse.xml"));
-
-			var client = GetWebApiClient();
-			var accessor = new SnapshotAccessor(client);
-
 			var filter = new SnapshotServicePlanListOptions();
 			filter.available = true;
 			filter.Id = Guid.NewGuid();
 
-			var pagingrequest = new PageableRequest();
-			pagingrequest.PageNumber = 1;
-			pagingrequest.PageSize = 5;
+			var pagingRequest = new PageableRequest();
+			pagingRequest.PageNumber = 1;
+			pagingRequest.PageSize = 5;
+
+			var expectedRelativeUriPath = string.Format(ApiUris.ListSnapshotServicePlans(accountId) +"?available={0}&id={1}&pageSize={2}&pageNumber={3}", "true", filter.Id, pagingRequest.PageSize, pagingRequest.PageNumber);
```

Comment:
Just added some more checks to ensure we at least asserting that the filters / options requested are sent to through to the end API URL we use when interacting with the Cloud Control API